tests/trx_generator/main.cpp

```diff
@@ -24,9 +24,11 @@ enum return_codes {
 };
 
 int main(int argc, char** argv) {
-   const int64_t TRX_EXPIRATION_MAX = 3600;
+   const int64_t trx_expiration_max = 3600;
+   const uint16_t generator_id_max = 960;
    variables_map vmap;
    options_description cli("Transaction Generator command line options.");
+   uint16_t gen_id;
```

Comment:
Should init to `0`. It doesn't look like boost reads the value, but better to be safe.

Reply:
Addressed: [f25de1b](https://github.com/AntelopeIO/leap/pull/702/commits/f25de1b5a8e69588160184bdd4b1316433a776c2)